starter-code/pizza.js

```diff
@@ -1 +1,54 @@
 // Write your Pizza Builder JavaScript in this file.
+
+function main() {
+
+
+  function bindIngredientClick(buttonSelector, ingridientSelector, ingridient) {
```

Comment:
Why haven't you used this function for every ingredient? The point of writing a function like this one is reuse it.